rackcity/utils/log_utils.py

```diff
@@ -0,0 +1,178 @@
+from datetime import datetime
+from django.contrib.auth.models import User
+from enum import Enum
+from rackcity.models import Log, Asset, ITModel, Datacenter
+
+
+class Action(Enum):
+    CREATE = "created"
+    MODIFY = "modified"
+    DELETE = "deleted"
+
+
+class PowerAction(Enum):
+    ON = "turned on"
+    OFF = "turned off"
+    CYCLE = "cycled"
+
+
+class ElementType(Enum):
+    ASSET = "asset"
+    DATACENTER = "datacenter"
+    MODEL = "model"
+    USER = "user"
+
+
+def datetime_to_string(date):
+    return "[" + str(date) + "]"
+
+
+def log_action(user, element, action):
+    """
+    Specified action should be Action enum.
+    """
+    date = datetime.now()
+    related_model = None
+    related_asset = None
+    if isinstance(element, Asset):
+        element_type = ElementType.ASSET.value
+        element_name = element.asset_number
+        related_asset = element
+    elif isinstance(element, ITModel):
+        element_type = ElementType.MODEL.value
+        element_name = element.vendor + " " + element.model_number
+        related_model = element
+    elif isinstance(element, User):
+        element_type = ElementType.USER.value
+        element_name = element.username
+    elif isinstance(element, Datacenter):
+        element_type = ElementType.DATACENTER.value
+        element_name = element.name
```

Comment:
Picky but I might prefer abbreviation here?  Would approve either way though

Reply:
i can dig it